test/Dapr.E2E.Test/DaprApp.cs

```diff
@@ -0,0 +1,99 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Dapr.E2E.Test
+{
+    public class DaprApp
+    {
+        // private string pathSeparator;
+        static string shellExeName = SetShell();
+        private string appId;
+        private int appPort;
+        const string outputToMatchOnStart = "You're up and running";
+        const string outputToMatchOnStop = "app stopped successfully";
+
+        static char pathSeparator = System.IO.Path.DirectorySeparatorChar;
```

Comment:
btw you can just do `using static System.IO.Path.DirectorySeparatorChar` and then reference `DirectorySeparatorChar`